### DISCUSSION TITLE
Chrome 0-days and Sandboxie

### DISCUSSION THREAD

**Discussion by Incog5** [sandboxie-plus/Sandboxie#2155] [General]
Quite a few zero days for chrome this year.  

1) Has anyone tested real samples of exploits with chrome + Sandboxie? 
2) Would Sandboxie isolate 0-day's that launches malicious code to the sandbox, and a firewall can protect data exfiltration?
3) Would start restrictions prevent any malicious code from launching in the Sandboxie in the first place?
4) Pro's vs Con's to running chrome in Sandboxie?

**Comment by DavidXanatos** [comment]
1. Not that I know of, it is not so easy to get one's hands on 0-days you know.
2. Sandboxie will contain the malicious code in the sandbox and yea a firewall or using a Sandboxie's build with WFP support will prevent data exfiltration to a certain extent. You see, if this box has exceptions set like allowing chrome to access the internet, malicious code can leverage the permitted processes to do the network communication on its behalf and a firewall will not prevent this. Hence, for optimal data protection, it is recommended to configure the box such that it will not have any access to sensitive data in the first place.
3. This depends on how this code runs, start restrictions will prevent new unpermitted processes form starting, it will not prevent a permitted sandboxed process from loading a malicious DLL, and it will also not do anything against code that is injected by other means into an otherwise permitted process. For the latter, the best protection is to have DEP enabled and Hardware-enforced Stack Protection if available.
4. Pro's, when Chrome will be compromised, as it inevitably will be at some point, attacker will be contained in the sandbox. Con's, there might be compatibility issues.
